src/components/container/sticky-bar/sticky-bar.scss

```diff
@@ -0,0 +1,78 @@
+@use '../../core/styles' as sbb;
+
+:host {
+  @include sbb.box-sizing;
+
+  --sbb-sticky-bar-padding-block: var(--sbb-spacing-responsive-xxxs);
+  --sbb-sticky-bar-animation-duration: var(--sbb-animation-duration-5x);
+  --sbb-sticky-bar-animation-easing: var(--sbb-animation-easing);
+  --sbb-sticky-bar-intersector-height: #{sbb.px-to-rem-build(1)};
+  --sbb-sticky-bar-background-color: var(
+    --sbb-container-background-color,
+    var(--sbb-color-white-default)
+  );
+
+  display: contents;
+}
+
+:host([color='white']) {
+  --sbb-sticky-bar-background-color: var(--sbb-color-white-default);
+}
+
+:host([color='milk']) {
+  --sbb-sticky-bar-background-color: var(--sbb-color-milk-default);
+}
+
+:host([color='midnight']:not([data-settled])) {
+  --sbb-sticky-bar-background-color: var(--sbb-color-midnight-default);
+  --sbb-focus-outline-color: var(--sbb-focus-outline-color-dark);
+
+  color: var(--sbb-color-white-default);
+}
+
+.sbb-sticky-bar__wrapper {
+  box-shadow: none;
+  position: sticky;
+  inset-block-end: 0;
+  display: block;
+  width: 100%;
+  background-color: var(--sbb-sticky-bar-background-color);
+  transition-property: box-shadow, background-color;
+  transition-duration: var(--sbb-sticky-bar-animation-duration);
+  transition-timing-function: var(--sbb-sticky-bar-animation-easing);
+
+  :host([data-sticking]) & {
+    @include sbb.shadow-level-5-hard;
+
+    @include sbb.if-forced-colors {
+      border-top: var(--sbb-border-width-1x) solid CanvasText;
+    }
+  }
+
+  :host(:not([data-sticking])) & {
+    --sbb-sticky-bar-background-color: var(
+      --sbb-container-background-color,
+      var(--sbb-color-white-default)
+    );
+  }
+}
+
+.sbb-sticky-bar {
+  display: flex;
+  padding-block: var(--sbb-sticky-bar-padding-block);
+
+  :host(:not([data-expanded])) & {
+    @include sbb.page-spacing;
+  }
+
+  :host([data-expanded]) & {
+    @include sbb.page-spacing-expanded;
+  }
+}
+
+.sbb-sticky-bar__intersector {
+  position: absolute;
+  width: 100%;
+  height: var(--sbb-sticky-bar-intersector-height);
+  transform: translateY(calc(var(--sbb-sticky-bar-intersector-height) * -1));
```

Comment:
Please write a short comment why the 1 pixel and transformation is necessary. Is it worth to have a css var for the intersector height, or could it be inlined as an internal detail?

Reply:
On windows the animation dind't work on any browser unless I gave the intersector an height. I'll document it with a comment. Inlined is fine to me.